pkg/splunk/controller/util.go

```diff
@@ -140,15 +140,6 @@ func MergePodSpecUpdates(current *corev1.PodSpec, revised *corev1.PodSpec, name
 				result = true
 			}
 
-			// check Env
-			if splcommon.CompareEnvs(current.Containers[idx].Env, revised.Containers[idx].Env) {
```

Comment:
basic question, why have we removed this?

Reply:
We don't need this check anymore, added when I was using configMap hash to update MC pos, now we use annotations